README.md

```diff
@@ -12,6 +12,4 @@ Source code for [blogpost on nojaf.com](https://blog.nojaf.com/react-hooks-in-fa
 
 - `yarn install`
 
-- `cd src`
-
-- `dotnet fable webpack-dev-server`
\ No newline at end of file
+- `npx webpack-dev-server`
```

Comment:
Would go with `yarn webpack-dev-server` since we are using yarn in this repo.

Reply:
Updated.